### ISSUE TITLE
DNS setup

### ISSUE THREAD

**Issue by 7yl4r** [marinebon/mbon-dashboard-server#30]
We need to set permanent URL for FWC, FGB, FK dashboards. 

@dotis : I can't find the email where we discussed this before. Do you recall what URL we want? Was it fwc-dashboard.marine.usf.edu ? We can do anything like `*.marine.usf.edu` pretty easily. `*.imars.marine.usf.edu` would be a little harder. `*.marinebon.org`, `*.usf.edu`, `*.ioos.us`, etc would invovle more red tape. We could also buy a domain to do something like `fwc.mbon-dashboards.com`.

Something like `*.marine.usf.edu` should be as easy as:

* [ ] reserve public IP in gcloud
* [ ] email doug w/ DNS setup request (as done with puppet)

Needs to be done for:

* [ ] FWC
* [ ] FGBNMS
* [ ] FKNMS

**Comment by dotis** [comment]
Yeah. I do remember an email. As I remember, we were going to use the easiest option, which I believe was something like:

FWC-dashboard.marine.usf.edu

and then:
FGBNMS-dashboard.marine.usf.edu
FKNMS-dashboard.marine.usf.edu

I think we just keep it simple and consistent.
________________________________
From: Tylar ***@***.***>
Sent: Thursday, June 10, 2021 1:24 PM
To: marinebon/mbon-dashboard-server ***@***.***>
Cc: dotis ***@***.***>; Mention ***@***.***>
Subject: [marinebon/mbon-dashboard-server] DNS setup (#30)


We need to set permanent URL for FWC, FGB, FK dashboards.

@dotis<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdotis&data=04%7C01%7Cdotis%40usf.edu%7C89dabf42fba24863743808d92c3495ca%7C741bf7dee2e546df8d6782607df9deaa%7C0%7C1%7C637589426621593279%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=wO6SO6rYzSbLoYrAgVhb85ABaG4fZoaUMGgxAN3WTiY%3D&reserved=0> : I can't find the email where we discussed this before. Do you recall what URL we want? Was it fwc-dashboard.marine.usf.edu ? We can do anything like *.marine.usf.edu pretty easily. *.imars.marine.usf.edu would be a little harder. *.marinebon.org, *.usf.edu, *.ioos.us, etc would invovle more red tape. We could also buy a domain to do something like fwc.mbon-dashboards.com.

Something like *.marine.usf.edu should be as easy as:

  *    reserve public IP in gcloud
  *    email doug w/ DNS setup request (as done with puppet)

Needs to be done for:

  *    FWC
  *    FGBNMS
  *    FKNMS

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmarinebon%2Fmbon-dashboard-server%2Fissues%2F30&data=04%7C01%7Cdotis%40usf.edu%7C89dabf42fba24863743808d92c3495ca%7C741bf7dee2e546df8d6782607df9deaa%7C0%7C1%7C637589426621603277%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=6CdmHfQYpwGovbuSoz3r0br43RDSm3n1doz50Lf3Y4U%3D&reserved=0>, or unsubscribe<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAGKS3DM44VOCZNKRI4SEIX3TSDYMHANCNFSM46O6KBXA&data=04%7C01%7Cdotis%40usf.edu%7C89dabf42fba24863743808d92c3495ca%7C741bf7dee2e546df8d6782607df9deaa%7C0%7C1%7C637589426621603277%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=IjX3VCDZ8bglhoUhkAS0CLr88TcmI6WnbWS9JxtlWrQ%3D&reserved=0>.

[EXTERNAL EMAIL] DO NOT CLICK links or attachments unless you recognize the sender and know the content is safe.

**Comment by dotis** [comment]
How soon can we have a permanent URL? like the simple marine.usf.edu one? Luke is asking. I think he needs to send the report on to DEP.


________________________________

**Comment by dotis** [comment]
Thanks. Do we know exactly what it will be?

http://FWC-dashboard.marine.usf.edu


http? https?
________________________________

**Comment by 7yl4r** [comment]
Yes, http://FWC-dashboard.marine.usf.edu is right. I will set up https
eventually (github issue #4
<https://github.com/marinebon/mbon-dashboard-server/issues/4>) but have not
prioritized it.
